### PR TITLE
validator badge collection

### PR DESCRIPTION
**Pull request by henry-zhang-bohan**
Add Validator Badge instructions.

### CODE REVIEW

**Comment by aaronmgdr** [comment]
are the badge files suppost to be markdown files? 
they are not viewable on gh so something seems wrong. 

also is web the best place for this?

File: packages/web/validator-badges/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+|006|Genesis - 1st Transaction|Validator Group address|
+|007|Celo Foundation - Vote Recipient|Validator Group address|
+
+Note:
+- If all validators in a group have Master Validator badge, then the validator group receives the Master Valdiator badge.
+- If at least 1 validator in a group is a Genesis Validator or a Stake Off Validator, then the group receives the badge.
+
+## How to Claim Your Validator Badge
+
+1. Append your Validator or Valdiator Group address (starting with `0x`) to the corresponding markdown files in this folder.
```
Comment (clrblmt) [inline packages/web/validator-badges/README.md]: LGTM just missing link to folder

**Review by clrblmt — APPROVED** [review]

**Comment by henry-zhang-bohan** [comment]
> are the badge files suppost to be markdown files?
> they are not viewable on gh so something seems wrong.
> 
> also is web the best place for this?

@aaronmgdr what do you think is the best place? I'm not sure what the best place would be tbh. Also, I think maybe they're not viewable because they're empty right now. Are there other concerns with markdown files?

**Comment by aaronmgdr** [comment]
> > are the badge files suppost to be markdown files?
> > they are not viewable on gh so something seems wrong.
> > also is web the best place for this?
> 
> @aaronmgdr what do you think is the best place? I'm not sure what the best place would be tbh. Also, I think maybe they're not viewable because they're empty right now. Are there other concerns with markdown files?

Id think this would be somewhere where it would be discoverable, like docs maybe? I just dont see how anyone would ever think to look for it here.

**Comment by henry-zhang-bohan** [comment]
> > > are the badge files suppost to be markdown files?
> > > they are not viewable on gh so something seems wrong.
> > > also is web the best place for this?
> > 
> > 
> > @aaronmgdr what do you think is the best place? I'm not sure what the best place would be tbh. Also, I think maybe they're not viewable because they're empty right now. Are there other concerns with markdown files?
> 
> Id think this would be somewhere where it would be discoverable, like docs maybe? I just dont see how anyone would ever think to look for it here.

If discoverablity is an issue, I can add it to the docs!

**Comment by aaronmgdr** [comment]
> > > > are the badge files suppost to be markdown files?
> > > > they are not viewable on gh so something seems wrong.
> > > > also is web the best place for this?
> > > 
> > > 
> > > @aaronmgdr what do you think is the best place? I'm not sure what the best place would be tbh. Also, I think maybe they're not viewable because they're empty right now. Are there other concerns with markdown files?
> > 
> > 
> > Id think this would be somewhere where it would be discoverable, like docs maybe? I just dont see how anyone would ever think to look for it here.
> 
> If discoverablity is an issue, I can add it to the docs!

makes sense to me

**Pull request merged**